cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -413,3 +414,23 @@ func DetectDefaultRouteInterface(cs *testclient.ClientSet, pod corev1.Pod) (stri
 	}
 	return "", fmt.Errorf("default route not present")
 }
+
+func GetStringEventsForPod(cs corev1client.EventsGetter, pod *corev1.Pod) string {
```

Comment:
nit: this function can be private to the package

Reply:
thanks, fixed